spec/features/orders_view_spec.rb

```diff
@@ -7,10 +7,12 @@
     @status = FactoryGirl.create(:status)
     @order = FactoryGirl.create(:order, user_id: @user.id, status: @status)
     @order2 = FactoryGirl.create(:order, status: @status)
-    visit login_path
-    fill_in("session_email", with: "paullorijgu@gmail.com")
-    fill_in("session_password", with: "gu")
-    click_button 'Sign in'
+    visit root_path
+    within("#signInModal") do
+      fill_in("session_email", with: "paullorijgu@gmail.com")
+      fill_in("session_password", with: "gu")
+      click_button 'Sign in'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.